mkdocs.yml

```
@@ -63,6 +63,7 @@ nav:
         - Using PostgreSQL: Tutorials/Using-PostgreSQL.md
         - TLS: Tutorials/TLS.md
         - Generating Orion certificates: Tutorials/Generating-Certificates.md
+        - Migrating to Tessera: Tutorials/Migrating-to-Tessera.md
```

```suggestion
        - Migrating from Orion to Tessera: Tutorials/Migrating-to-Tessera.md
```
and maybe rename the file to `Migrating-from-Orion-to-Tessera.md` too.